addons/mail/static/src/core/common/thread.xml

```diff
@@ -102,14 +98,23 @@
 </t>
 
 <t t-name="mail.Thread.jumpPresent">
-    <span t-if="props.showJumpPresent and state.showJumpPresent" class="o-mail-Thread-jumpPresent position-sticky btn btn-link alert alert-info d-flex cursor-pointer align-items-center py-2 m-0" t-att-class="{ 'px-4': !env.inChatWindow, 'px-2': env.inChatWindow, 'top-0': props.order !== 'asc', 'bottom-0': props.order === 'asc' }" role="button" t-on-click="() => this.jumpToPresent()">
-        <span class="small">You're viewing older messages</span>
-        <span class="flex-grow-1"/>
-        <span class="fw-bolder small pe-2">Jump to Present</span>
-        <i class="fa" t-att-class="{ 'fa-caret-up': props.order !== 'asc', 'fa-caret-down': props.order === 'asc' }"/>
+    <span  t-if="props.showJumpPresent and state.showJumpPresent" t-att-class="{
+        'mx-5 rounded-top': !env.inChatWindow and !env.inChatter,
+        'm-0 px-4 position-sticky top-0': env.inChatter
+    }" class="o-mail-Thread-banner d-flex justify-content-between alert alert-primary border-0 rounded-0 mb-0 py-1 cursor-pointer shadow-sm small fw-bold" t-on-click="() => this.jumpToPresent()">
+        <span>You're viewing older messages</span>
+        <span>Jump to Present<i class="ms-2 fa" t-att-class="{ 'fa-caret-up': props.order !== 'asc', 'fa-caret-down': props.order === 'asc' }"/></span>
     </span>
 </t>
 
+<t t-name="mail.Thread.jumpToUnread">
+    <span t-if="props.thread.selfMember?.localMessageUnreadCounter > 0" class="d-flex cursor-pointer shadow-sm small fw-bold" t-att-class="{'mx-5': !env.inChatWindow}">
+        <t t-set="alertClass" t-value="'alert alert-info m-0 o-mail-Thread-banner rounded-top-0 py-1'"/>
+        <t t-set="alertAttClass" t-value="{'rounded-0': env.inChatWindow}"/>
+        <span t-att-class="alertAttClass" t-attf-class="{{ alertClass }} rounded-end-0 border-start-0 flex-grow-1" t-on-click="onClickUnreadMessagesBanner"><t t-esc="props.thread.selfMember?.localMessageUnreadCounter"/> new messages</span>
+        <span t-att-class="alertAttClass" t-attf-class="{{ alertClass }} rounded-start-0 border-start" t-on-click="() => props.thread.markAsRead({ sync: true })">Mark as Read<i class="ms-2 fa fa-check-square"/></span>
+    </span>
+</t>
```

Comment:
The banners should take whole width also in discuss app.
![discuss-app](https://github.com/odoo/odoo/assets/6569390/a345494e-ccbd-4ef3-bc30-84e0b38139c0)

Reply:
I also think the UI is too noisy with the 2 banners. Having "Jump to present" as a banner made sense when there was not the unread banner. But we could turn it into just a floating arrow. Sure this is less explicit, but any users now what a floating arrow implies, so this would reduce the bloat on UI.

<img width="981" alt="floating-arrow" src="https://github.com/odoo/odoo/assets/6569390/63c0d330-06d8-4668-8a58-bc879adc5076">

@paju-odoo What do you think?

Not a huge deal, it works with 2 banners. I don't mind to think of such improvements later to not delay this task further :)